incubating/aws-s3/step.yaml

```diff
@@ -0,0 +1,101 @@
+kind: step-type
+version: '1.0'
+metadata:
+  name: aws-s3
+  version: 1.0.0
+  isPublic: true
+  description: Integration that uploads files to S3
+  sources:
+    - https://github.com/codefresh-io/steps/tree/master/incubating/aws-s3
+  stage: incubating
+  maintainers:
+    - name: Luke Goodfellow
+    - email: luke.goodfellow@codefresh.io
+  categories:
+    - utilities
+  official: true
+  tags: []
+  icon:
+    type: svg
+    url: https://raw.githubusercontent.com/codefresh-io/steps/aws-s3/incubating/aws-cdk/images/s3-logo.svg
+    background: "#f4f4f4"
+  examples:
+    - description: aws-s3
+      workflow:
+        uploadToS3:
+          title: Uploading to S3
+          type: aws-s3
+          stage: cleanup
+          arguments:
+            REGION: us-east-1
+            BUCKET: my-s3-bucket
+            S3_PREFIX: myDirectory/subdirectory
+            S3_INTEGRATION: amazon
+            SOURCE: ${{CF_REPO_NAME}}/myDirectory
+spec:
+  arguments: |-
+    {
+      "definitions": {},
+      "$schema": "http://json-schema.org/draft-07/schema#",
+      "type": "object",
+      "additionalProperties": true,
+      "patterns": [],
+      "required": ["BUCKET", "S3_PREFIX", "S3_INTEGRATION", "SOURCE"],
+      "properties":
+        {
+          "S3_INTEGRATION": {
+            "type": "string",
+            "description": "The name of the S3 storage integration"
+            },
+          "AWS_IMAGE": {
+            "type": "string",
+            "default": "quay.io/codefreshplugins/aws-s3",
+            "description": "The AWS S3 container image registry/image for the step."
+            },
+          "AWS_IMAGE_VERSION": {
+            "type": "string",
+            "default": "1.0.0",
+            "description": "Version of the AWS S3 image to use, Docker image tag."
+            },
+          "REGION": {
+              "type": "string",
+              "description": "The region the bucket is in. Default is us-east-1",
+              "default": "us-east-1"
+            },
+          "BUCKET": {
+              "type": "string",
+              "description": "The name of the bucket you are using"
+            },
+          "S3_PREFIX":
+            {
+              "type": "string",
+              "description": "The directory in S3 you want the files to be uploaded to"
```

Comment:
if the name does not end by '/' that will be the name of the file (for a single file upload)
Maybe put some kind of note

Reply:
Updated the wording